charts/rancher-external-ip-webhook/rancher-external-ip-webhook/100.0.0+up0.1.6/templates/NOTES.txt

```diff
@@ -1,3 +0,0 @@
-To verify that externalip-webhook has started, run:
```

Comment:
The charts and corresponding archive file for rancher-external-ip-webhook are removed.